sway/tree/root.c

```diff
@@ -56,6 +56,17 @@ void root_destroy(struct sway_root *root) {
 	free(root);
 }
 
+static void set_container_transform(struct sway_workspace *ws,
+			struct sway_container *con) {
+	struct sway_output *output = ws->output;
+	if (!output) {
+		output = root->fallback_output;
```

Comment:
Maybe just explicitly set the transform to zeros. I'm not sure the fallback output is sensible here.